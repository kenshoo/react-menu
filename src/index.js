export { Menu } from "./components/menu";
