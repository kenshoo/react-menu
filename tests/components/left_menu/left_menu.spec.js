import React from "react";
import renderer from "react-test-renderer";

import { LeftMenu } from "../../../src/";

describe("LeftMenu component", () => {
  test("renders children", () => {
    const cmp = renderer.create(<LeftMenu>TEST</LeftMenu>);
    expect(cmp).toMatchSnapshot();
  });

  test("recieves className", () => {
    const cmp = renderer.create(
      <LeftMenu className="TEST_CLASS">TEST</LeftMenu>
    );
    expect(cmp).toMatchSnapshot();
  });
});
