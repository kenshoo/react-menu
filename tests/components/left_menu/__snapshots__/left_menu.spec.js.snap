// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LeftMenu component recieves className 1`] = `
<div
  className="left_menu TEST_CLASS"
>
  TEST
</div>
`;

exports[`LeftMenu component renders children 1`] = `
<div
  className="left_menu"
>
  TEST
</div>
`;
